extension/statistics.js

```diff
@@ -1,36 +1,45 @@
 const STATS_PATTERNS = [{
         name: "stable",
         pattern: null,
+        number: 1,
     },
     {
         name: "nightly",
         pattern: /^\/[^/].*/i,
+        number: 2,
     },
     {
         name: "docs.rs",
         pattern: /^[~@].*/i,
+        number: 3,
     },
     {
         name: "crate",
         pattern: /^!!!.*/i,
+        number: 4,
     },
     {
         name: "attribute",
         pattern: /^#.*/i,
+        number: 5,
     },
     {
         name: "error code",
         pattern: /^`?e\d{2,4}`?$/i,
+        number: 6,
     },
     {
         name: "rustc",
         pattern: /^\/\/.*/i,
+        number: 7,
     },
     {
-        name: 'other',
+        name: "other",
         pattern: /^[>%?]|(v?1\.).*/i,
+        number: 8,
```

Comment:
I suggest setting the number of `other` to `999`, giving enough space for future types.